src/main/solution_space/path_finder_test_system.py

```diff
@@ -27,13 +27,15 @@
 from src.main.solution_space.path_finder.path_finder import IPathFinder
 from src.main.canonicalization.canonicalization import get_code_from_tree
 from src.main.solution_space.measured_tree.measured_tree import IMeasuredTree
+from src.main.util.consts import LOGGER_NAME, INT_EXPERIENCE, TEST_RESULT, TASK
 from src.main.solution_space.solution_space_serializer import SolutionSpaceSerializer
 from src.main.solution_space.solution_space_visualizer import SolutionSpaceVisualizer
 
 log = logging.getLogger(LOGGER_NAME)
 
 
 class TEST_INPUT(Enum):
+    INDEX = 'index'
```

Comment:
Maybe we don't need INDEX here? You change it to  `i` anyway at line 137:
`test_input[TEST_INPUT.INDEX] = i`
so it doesn't store any information.
So according to realization there is no need to pass INDEX together with other test inputs, that's also a reason not to include INDEX in TEST_INPUT.